src/supporting-data/citations/config/CitationsColumnConfiguration.tsx

```diff
@@ -0,0 +1,125 @@
+import { Link } from 'react-router-dom';
+
+import { getEntryPathFor } from '../../../app/config/urls';
+
+import {
+  CitationsAPIModel,
+  CitationXRefDB,
+} from '../adapters/citationsConverter';
+import { ColumnConfiguration } from '../../../shared/types/columnConfiguration';
+import { JournalInfo } from '../../../shared/components/literature-citations/LiteratureCitation';
+import { Namespace } from '../../../shared/types/namespaces';
+
+export enum CitationsColumn {
+  // TODO: Report to backend that this field query is not working
+  authoringGroup = 'authoring_group',
+  doi = 'doi',
+  firstPage = 'first_page',
+  id = 'id', // Pubmed ID
+  journal = 'journal',
+  lastPage = 'last_page',
+  litAbstract = 'lit_abstract',
+  publicationDate = 'publication', // ⚠️ field doesn't contain the string "date"
+  // This map to a combination of all the fields to make a citation reference
+  reference = 'reference',
+  statistics = 'statistics',
+  title = 'title',
+  volume = 'volume',
+}
+
+// TODO: review
+export const defaultColumns = [
+  CitationsColumn.id,
+  CitationsColumn.title,
+  CitationsColumn.authoringGroup,
+  CitationsColumn.publicationDate,
+  CitationsColumn.journal,
+  CitationsColumn.firstPage,
+  CitationsColumn.lastPage,
+];
+
+export const primaryKeyColumn = CitationsColumn.id;
+
+const getEntryPath = getEntryPathFor(Namespace.citations);
+
+export const CitationsColumnConfiguration: ColumnConfiguration<
+  CitationsColumn,
+  Partial<CitationsAPIModel>
+> = new Map();
+
+// COLUMN RENDERERS BELOW
+CitationsColumnConfiguration.set(CitationsColumn.authoringGroup, {
+  label: 'Authors',
+  render: ({ citation }) => citation?.authors?.join(', '),
+});
+
+CitationsColumnConfiguration.set(CitationsColumn.doi, {
+  label: 'DOI',
+  render: ({ citation }) =>
+    citation?.citationCrossReferences?.find(
+      (xref) => xref.database === CitationXRefDB.DOI
+    )?.id,
+});
+
+CitationsColumnConfiguration.set(CitationsColumn.firstPage, {
+  label: 'First page',
+  render: ({ citation }) => citation?.firstPage,
+});
+
+CitationsColumnConfiguration.set(CitationsColumn.id, {
+  label: 'PubMed ID',
```

Comment:
Yep, I find it strange that we use the PubMed ID as internal citation ID. Would need to have a chat about that and if it is the case always, because the way it was in the data made me feel like there could be cases where there might not be a PubMed ID for a citation, because this is not clearly labeled as an ID (because the PubMed ID is in an object, part of an array, instead of being at the top level).
Need a chat with backend about that, and see if we can expose the UniProt citation ID alongside the PubMed ID, even if they are the same, but at least to have 2 different columns